src/parser/jobs/pld/modules/Gauge.tsx

```diff
@@ -0,0 +1,68 @@
+import {t} from '@lingui/macro'
+import {Trans} from '@lingui/react'
+import {ActionLink} from 'components/ui/DbLink'
+import ACTIONS from 'data/ACTIONS'
+import JOBS from 'data/JOBS'
+import {dependency} from 'parser/core/Module'
+import {CounterGauge, Gauge as CoreGauge} from 'parser/core/modules/Gauge'
+import Suggestions, {SEVERITY, Suggestion} from 'parser/core/modules/Suggestions'
+import React from 'react'
+
+// Gauge numbers
+const GAUGE_GAIN_AMOUNT = 5
+const GAUGE_SPEND_AMOUNT = 50
```

Comment:
Might be cleaner to make this negative rather than inverting it in the gauge adjustment, it's easier to read and clearer to maintain.

Reply:
I'll remove it in favor of this change https://github.com/xivanalysis/xivanalysis/pull/1153#discussion_r621833815